src/RawHID.h

```diff
@@ -23,10 +20,11 @@ THE SOFTWARE.
 */
 #pragma once
 
-#include <Arduino.h>
+#include "Arduino.h"
+#include "CorsairLightingProtocolConstants.h"
```

Comment:
I don't like this reference here, RawHID should be decoupled from the Corsair specific stuff

Reply:
The issue was the serial number going into the constants. I use it in TinyUSBHID too. I didn't think we wanted to have it in two places. What do you want to do with it?